src/Model/GServer.php

```diff
@@ -2063,4 +2064,19 @@ public static function getProtocol(int $gsid): ?int
 
 		return null;
 	}
+
+	/**
+	 * Enforces gserver table field maximum sizes to avoid "Data too long" database errors
+	 *
+	 * @param array $fields
+	 * @param array $condition
+	 * @return bool
+	 * @throws Exception
+	 */
+	public static function update(array $fields, array $condition): bool
+	{
+		$fields = DBStructure::getFieldsForTable('post-thread', $fields);
```

Comment:
You should use "gserver" and not "post-thread", because you want to match against the definition of that table.

Reply:
Copy-paste issue, you can relate!